src/main/kotlin/no/nav/familie/ef/sak/vedtak/domain/Vedtak.kt

```diff
@@ -24,6 +26,11 @@ data class Vedtak(@Id
                   val saksbehandlerIdent: String? = null,
                   @Column("opphor_fom")
                   val opphørFom: LocalDate? = null,
+                  val barnetilsyn: BarnetilsynWrapper? = null,
+                  @Column("kontantstotte")
+                  val kontantstøtte: KontantstøtteWrapper? = null,
+                  @Column("tilleggsstonad")
+                  val tilleggsstønad: TilleggsstønadWrapper? = null,
```

Comment:
Betyr dette at feltet `perioder` bare brukes for overgangsstønad?

Reply:
Burde vi ha en OvergangsstønadWrapper også?

Reply:
@eirarset Ja, tror nok det. For Barnetilsyn har ingen begrep om periodetype og aktivitetstype - så blir litt rart å blande det sammen på dette nivået iallefall. 

@ma10s Ja, men det krever vel at vi patcher alle verdiene som ligger i databasen allerede. Det må isåfall være en egen PR. Lager en forvaltningsoppgave på det. https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=Tea-8391